src/main/webapp/style.css

```diff
@@ -1,38 +1,39 @@
-* {
-  font-family: 'Product Sans';
-}
+/* Global styling */
 
-body {
-  margin-bottom: 60px !important;
-}
-
-html {
-  min-height: 100%;
-  position: relative;
+@font-face {
```

Comment:
For this whole file I would look into alphabetizing all the declarations. 
Google code is usually arranged in alphabetical order for every kind of declaration and this is done recursively.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
We will organize/split CSS in a future PR, if that's ok

Reply:
Why not split it now?

Reply:
There are a lot of other PRs out rn that touch this file and it would complicate a bunch of merges.  I could create a new PR with everything split, but we'd have to redo all of the styling merges again...  I was gonna make the whole directory restructure it's own PR

Reply:
Tracking issue #123